release/cluster_tests/aws_cluster.yaml

```diff
@@ -0,0 +1,16 @@
+# An unique identifier for the head node and workers of this cluster.
+cluster_name: nightly-test-minimal
+max_workers: 0
+idle_timeout_minutes: 2
+
+# Cloud-provider specific configuration.
+provider:
+    type: aws
+    region: us-west-2
+    cache_stopped_nodes: False
+
+available_node_types:
+    ray.head.default:
+        resources: {}
+        node_config:
+            InstanceType: t3.large
```

Comment:
This is the OSS yaml right? Why don't we test the example config we have? For example, 

https://github.com/ray-project/ray/blob/master/python/ray/autoscaler/aws/example-ml.yaml

Reply:
a bit too expensive, also we need unique name :)